mobile/navigation/index.tsx

```diff
@@ -388,48 +395,28 @@ function BookNavigator() {
           name="BabyBookAccess"
           component={BabyBookAccess}
           options={{
-            headerTitle: () => (
-              // add progress bar/circles and styling here
-              <View>
-                <Text>Baby Book Access</Text>
-              </View>
-            ),
+            headerShown: false,
           }}
         />
         <Book.Screen
           name="StartBook"
           component={StartBook}
           options={{
-            headerTitle: () => (
-              // add progress bar/circles and styling here
-              <View>
-                <Text>Start A Baby Book</Text>
-              </View>
-            ),
+            header: () => <View></View>,
```

Comment:
Is this needed? I see it's all over, so I suspect it might be.

Reply:
The default header has a back button & title, this removes the title and keeps the back button. There's probably a better way to remove the title though 😅